process/elasticProcessor.go

```diff
@@ -313,7 +319,27 @@ func (ei *elasticProcessor) RemoveTransactions(header coreData.HeaderHandler, bo
 		return nil
 	}
 
-	return ei.elasticClient.DoBulkRemove(elasticIndexer.TransactionsIndex, encodedTxsHashes)
+	return ei.elasticClient.DoQueryRemove(
+		elasticIndexer.TransactionsIndex,
+		converters.PrepareHashesForQueryRemove(encodedTxsHashes),
+	)
+}
+
+// RemoveAccountsESDT will remove data from accountsesdt index and accountsesdthistory
+func (ei *elasticProcessor) RemoveAccountsESDT(headerTimestamp uint64) error {
+	query := fmt.Sprintf(`{"query": {"bool": {"must": [{"match": {"shardID": {"query": %d,"operator": "AND"}}},{"match": {"timestamp": {"query": "%d","operator": "AND"}}}]}}}`, ei.selfShardID, headerTimestamp)
```

Comment:
there is still a `"query": %d` is it ok?

Reply:
yes is ok.